src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/Input/CommandGroup.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.Input
+{
+    internal static class CommandGroup
+    {
+        public const string ManagedProjectSystemClientProjectEdit = "{28C12D02-11CB-437D-B84D-9CEA7A5333A3}";
```

Comment:
Corresponds to `guidManagedProjectSystemClientProjectEditCommandSet` in the .vsct file.